openstack/passwords.go

```diff
@@ -0,0 +1,31 @@
+package openstack
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/hashicorp/vault/sdk/helper/base62"
+)
+
+const (
+	PasswordLength = 16
+)
+
+type PasswordGenerator interface {
+	GeneratePasswordFromPolicy(ctx context.Context, policyName string) (password string, err error)
+}
+
+type Passwords struct {
+	PolicyGenerator PasswordGenerator
+	PolicyName      string
```

Comment:
Why is it called `Name` ― per documentation it's a template

Reply:
```go
Description: "Name of the password policy to use to generate passwords for dynamic credentials.",
```